core/domain/user_services.py

```diff
@@ -531,6 +531,59 @@ def get_user_role_from_id(user_id):
     return user_settings.role
 
 
+def get_user_community_rights(user_id):
+    """Returns the UserCommunityRights for the given user_id.
+
+    Args:
+        user_id: str. The unique Id of the user.
+
+    Returns:
+        UserCommunityRights. The UserCommunityRights domain object for the
+        corresponding user.
+    """
+    user_community_rights_model = (
+        user_models.UserCommunityRightsModel.get_by_id(user_id))
+    if user_community_rights_model is not None:
+        return user_domain.UserCommunityRights(
+            user_id,
+            user_community_rights_model.can_review_translation_in_languages,
+            user_community_rights_model.can_review_voiceover_in_languages,
+            user_community_rights_model.can_review_questions)
+    else:
+        return user_domain.UserCommunityRights(user_id, [], [], False)
+
+
+def get_all_community_reviewers():
```

Comment:
Is there any upper limit?

I'm wondering if any paging is needed -- if not, we might want to enforce a limit of "don't add more community reviewers than N" somewhere.